rocblas/PKGBUILD

```diff
@@ -19,15 +19,16 @@ build() {
   export CFLAGS="$(sed -e 's/-fstack-protector-strong//' <<< "$CFLAGS")"
   export CXXFLAGS="$(sed -e 's/-fstack-protector-strong//' <<< "$CXXFLAGS")"
   export CPPFLAGS="$(sed -e 's/-fstack-protector-strong//' <<< "$CPPFLAGS")"
-
+  
+  PATH="/opt/rocm/llvm/bin:${PATH}" \
```

Comment:
I'm sure there is a better way of doing this. Like overriding a certain CMAKE flag to specify the clang path. For now let's just go with it.

Reply:
I don't like it either. But Tensile has its own `which` implementation in python which first searches in `PATH`. The other possibility would be to patch the python code.